Assets/MixedRealityToolkit/MixedRealityToolkit.asmdef

```diff
@@ -2,7 +2,8 @@
     "name": "Microsoft.MixedReality.Toolkit",
     "references": [
         "Microsoft.MixedReality.Toolkit.Core.Utilities.Editor",
-        "Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities.Editor"
+        "Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities.Editor",
+        "Microsoft.MixedReality.Toolkit.Core.Utilities.Editor.Setup"
```

Comment:
Does this assembly need to exist? It appears to contain a single file that would otherwise be captured by the `Microsoft.MixedReality.Toolkit.Core.Utilities.Editor` assembly (which is also an editor-only assembly).

Reply:
Yes. Absolutely. Or else the rest of them wouldn't work.

Reply:
I'm not asking "does it need to be added to this list?". I'm asking "does this separate assembly need to exist at all?".

Reply:
If it didn't exist, then the rest wouldn't work.

Not much to explain.

Reply:
Why wouldn't the rest work?

Reply:
Go read the code and find out